packages/web/components/Primitives/Styled.js

```diff
@@ -0,0 +1,180 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  //Base
+  space,
+  width,
+  color,
+  fontSize,
+  fontWeight,
+  fontFamily,
+  textAlign,
+  lineHeight,
+  letterSpacing,
+  display,
+  maxWidth,
+  minWidth,
+  height,
+  maxHeight,
+  minHeight,
+  size,
+  verticalAlign,
+  // Grid
+  gridGap,
+  gridRowGap,
+  gridColumnGap,
+  gridColumn,
+  gridRow,
+  gridArea,
+  gridAutoFlow,
+  gridAutoRows,
+  gridAutoColumns,
+  gridTemplateRows,
+  gridTemplateColumns,
+  gridTemplateAreas,
+  //Flex
+  alignItems,
+  justifyContent,
+  flexWrap,
+  flexDirection,
+  flex,
+  alignContent,
+  justifyItems,
+  justifySelf,
+  alignSelf,
+  order,
+  flexBasis,
+  //background
+  background,
+  backgroundImage,
+  backgroundSize,
+  backgroundPosition,
+  backgroundRepeat,
+  //Misc,
+  border,
+  borderTop,
+  borderRight,
+  borderBottom,
+  borderLeft,
+  borderWidth,
+  borderStyle,
+  borderColor,
+  borderRadius,
+  boxShadow,
+  opacity,
+  overflow,
+  borders,
+  // Position
+  position,
+  zIndex,
+  top,
+  right,
+  bottom,
+  left,
+  // Misc
+  buttonStyle,
+  variant
+} from "styled-system";
```

Comment:
I liked the concept, it's going to work with linaria too right?

Reply:
Yeah! Works with any Styled component like library

Reply:
once we fix the Parcel plugin for linaria we can move back to Linaria